js/huobi.js

```diff
@@ -2751,41 +2783,38 @@ module.exports = class huobi extends Exchange {
         //
         const data = this.safeValue (response, 'data', []);
         const result = {};
+        this.networkJunctionsByTitles = {};
```

Comment:
I think we should avoid defining them as class props. Derived class properties are not transpilable, so we would need to add them to the base which is not desirable because this is specific to huobi. 

Even though this might be working now, it won't work after we migrate to TS or another language so can't we add them to options?

Reply:
> Derived class properties are not transpilable

I fear that I don't fully understand, what means are not transpilable? do you mean that they are not being un_camel_cased? otherwise, i see that they are being transpiled

Reply:
@ttodua no, in a properly typed language you need to declare the property before using it, and we're not doing it here and we can't really because class properties are not transpilable

Reply:
ah, i understand what you mean. you mean that the declarations (of properties) are not being transpiled, right, ofc (i though that `this.whateverProperty` occasions in method bodies were not being transpiled : ) 

and right, in ideal cases they should be declared in class. but populating `this.options` with 1MB data doesn't seem good to me. can we implement something in base, like `this.instanceOptions` or whatever, which will take that functionality, so `options` will stay purely "read-only" like or much less mutated.

Reply:
@ttodua 
>  but populating this.options with 1MB data doesn't seem good to me

why would be different than storing it in a property?

Reply:
The reason is just readability. I believe that `options` should be a property which (should) hold the true options(settings) set by user , but all other variables and data that live just during lifespan of instance, should be held in other instance property. However, that is just my optimal design, and as you say, in this case, I'll do as you suggest and will move them in options.